Gliocas/gliocas_app/models.py

```diff
@@ -51,11 +53,11 @@ class Question(models.Model):
     text = models.TextField(max_length=textLength)
     date = models.DateTimeField(null=True)
     views = models.IntegerField(default=0)
-    slug = models.SlugField(unique=True)
+    slug = models.SlugField(max_length=8, default=get_random_string(length=9), unique=True)
```

Comment:
this makes Django run get_random_string(length=9) once at beginning of migration, and then all questions get the same slug, so UNIQUE constraint fails (did you not try running populate.py?) remove (length=9) so it uses the function to get a new value everytime instead.

```suggestion
    slug = models.SlugField(max_length=8, default=get_random_string, unique=True)
```